vignettes/ggcoef_model.Rmd

```diff
@@ -0,0 +1,258 @@
+---
+title: "ggcoef_model(): Model coefficients"
+output: rmarkdown::html_vignette
+author: Joseph Larmarange
+date: May 16, 2016
+vignette: >
+  %\VignetteIndexEntry{ggcoef_model(): Model coefficients}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
```

Comment:
Don't forget to add this in the `articles` section of `./_pkgdown.yml`

Reply:
already done